### DISCUSSION TITLE
Can't share application on Transporter (iOS)

### DISCUSSION THREAD

**Discussion by ZhorovEgor** [airsdk/Adobe-Runtime-Support#3520] [Q&A]
Good evening, I had a problem with share my app. I'd make all what need on this [instruction](https://docs.airnativeextensions.com/docs/tutorials/ios-launchscreens/) but i'v got this problem. 
<img width="839" alt="Снимок экрана 2024-10-12 в 20 55 54" src="https://github.com/user-attachments/assets/ff836902-81e4-403f-871f-2dbcf1a44755">

This is my steps - 
<img width="1280" alt="Снимок экрана 2024-10-12 в 20 48 02" src="https://github.com/user-attachments/assets/aa704fe0-2772-4e6e-be0f-78f8a92e4cd5">
<img width="1280" alt="Снимок экрана 2024-10-12 в 20 48 09" src="https://github.com/user-attachments/assets/1cb1ac74-d959-4999-a89d-d7ca013b4e80">

**Comment by FliplineStudios** [comment]
Hmm, and you definitely packaged the Assets.car in the IPA?  (You can change .ipa to .zip, and unzip it to make sure it's there in Payload > Yourapp.app)  And you included both the 152px and 1024px images in Assets.car?  

The first error sounds like it's related to the pngs instead of Assets.car, did you add `<image152x152>` into the `<icon>` section of your application xml with a png?  You should also see the png version in your unzipped .ipa titled "Icon-Ipad-152.png" (which is where AIR puts that png you specified in the xml)

And one final thought -- when you were setting up icons in Assets.car, did you use the newer method of just supplying one 1024 image in Xcode rather than each individual image?  Or did you use the previous method (like in that linked tutorial) where you add each separate image at each size?  I remember at one point there were validation issues on Apple's end if you only added the one image to the newer Xcode instead of all of separate ones, I'm not sure if they ever resolved that.

**Comment by ghost** [comment]
Check out the last comment

https://forum.starling-framework.org/d/20366-missing-required-icon-file/19
